changelog.txt

```diff
@@ -1,3 +1,26 @@
+3.0.1-preview
+=============
+New Features:
+- Added FOCI support
+- Device Code supports both verification_url and verification_uri
+- MsalError contains all the error messages
+- MsalException and its derived exception can now be serialized to JSon and deserialized
+- MSAL.NET for .NET Core moved to .NET Core 2.1.
+- At both the app creation and the token acquisition, you can now pass extra query parameters as a string (in addition to a Dictionary<string,string> introduced in MSAL 3.0.0
+- MSAL.NET symbols are now published on the Microsoft Symbol server, which is useful when you are debugging.
```

Comment:
This is build improvement. Better mention SourceLink instead (also a build improvement, but with more customer value)